thorlcr/master/thgraphmanager.cpp

```diff
@@ -1095,6 +1095,38 @@ void closeThorServerStatus()
     }
 }
 
+
+#ifdef _CONTAINERIZED
+static int recvNextGraph(unsigned timeoutMs, const char *wuid, StringBuffer &retWuid, StringBuffer &retGraphName)
```

Comment:
Worth having a comment about the meaning of the return codes

Reply:
will add